benchmark/README.md

```diff
@@ -121,50 +122,51 @@ Specs: [details](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/spe
+5.96       5.93       5.90       [160, 120]   YuNet with ['face_detection_yunet_2023mar.onnx']
+6.09       6.11       5.90       [160, 120]   YuNet with ['face_detection_yunet_2023mar_int8.onnx']
+73.30      73.22      72.32      [150, 150]   SFace with ['face_recognition_sface_2021dec.onnx']
+88.20      89.95      72.32      [150, 150]   SFace with ['face_recognition_sface_2021dec_int8.onnx']
+32.33      32.20      31.99      [112, 112]   FacialExpressionRecog with ['facial_expression_recognition_mobilefacenet_2022july.onnx']
+39.82      40.78      31.99      [112, 112]   FacialExpressionRecog with ['facial_expression_recognition_mobilefacenet_2022july_int8.onnx']
+108.37     108.31     106.93     [224, 224]   MPHandPose with ['handpose_estimation_mediapipe_2023feb.onnx']
+75.91      78.95      49.78      [224, 224]   MPHandPose with ['handpose_estimation_mediapipe_2023feb_int8.onnx']
+76.29      77.10      75.21      [192, 192]   PPHumanSeg with ['human_segmentation_pphumanseg_2023mar.onnx']
+77.33      77.73      75.21      [192, 192]   PPHumanSeg with ['human_segmentation_pphumanseg_2023mar_int8.onnx']
+66.22      66.09      65.90      [224, 224]   MobileNet with ['image_classification_mobilenetv1_2022apr.onnx']
+59.91      60.72      54.63      [224, 224]   MobileNet with ['image_classification_mobilenetv2_2022apr.onnx']
+62.83      54.85      54.63      [224, 224]   MobileNet with ['image_classification_mobilenetv1_2022apr_int8.onnx']
+62.47      62.13      54.63      [224, 224]   MobileNet with ['image_classification_mobilenetv2_2022apr_int8.onnx']
+625.82     667.05     425.55     [224, 224]   PPResNet with ['image_classification_ppresnet50_2022jan.onnx']
+508.92     667.04     373.14     [224, 224]   PPResNet with ['image_classification_ppresnet50_2022jan_int8.onnx']
+147.19     146.62     146.31     [320, 240]   LPD_YuNet with ['license_plate_detection_lpd_yunet_2023mar.onnx']
+143.70     155.87     139.90     [320, 240]   LPD_YuNet with ['license_plate_detection_lpd_yunet_2023mar_int8.onnx']
+214.87     214.19     213.21     [416, 416]   NanoDet with ['object_detection_nanodet_2022nov.onnx']
+212.90     212.93     209.55     [416, 416]   NanoDet with ['object_detection_nanodet_2022nov_int8.onnx']
+1690.06    2303.34    1480.63    [640, 640]   YoloX with ['object_detection_yolox_2022nov.onnx']
+1489.54    1435.48    1308.12    [640, 640]   YoloX with ['object_detection_yolox_2022nov_int8.onnx']
+564.90     580.35     527.49     [1280, 720]  DaSiamRPN with ['object_tracking_dasiamrpn_kernel_cls1_2021nov.onnx', 'object_tracking_dasiamrpn_kernel_r1_2021nov.onnx', 'object_tracking_dasiamrpn_model_2021nov.onnx']
+356.63     357.29     354.42     [192, 192]   MPPalmDet with ['palm_detection_mediapipe_2023feb.onnx']
+217.52     229.39     101.61     [192, 192]   MPPalmDet with ['palm_detection_mediapipe_2023feb_int8.onnx']
+198.63     198.25     196.68     [224, 224]   MPPersonDet with ['person_detection_mediapipe_2023mar.onnx']
```

Comment:
These models (`MPPalmDet`, `MPPersonDet` and `MPHandPose) run significantly slower than before on rasberry pi. Do you know why? @WanliZhong

Reply:
I'm not sure, let me test it layer by layer.

Reply:
Partly fixed in https://github.com/opencv/opencv/pull/23952#issuecomment-1628341286 . Palm detection is still a little slower than 4.7 may due to layer fusion problem, which need to be discussed.